src/models/GenericOrder.ts

```diff
@@ -1,6 +1,6 @@
-import { OrderState } from './OrderState';
-import { OrderType } from './OrderType';
-import { ProjectAllocation } from './ProjectAllocation';
+import type { OrderState } from './OrderState.js';
```

Comment:
Why do these need `.js`? It seems counterintuitive since we're importing a type.